src/IW/Time.hs

```diff
@@ -0,0 +1,18 @@
+module IW.Time
+       ( firstHaskellRepoCreated
+       , getToday
+       , julianDayToIso
+       ) where
+
+import Data.Time (Day (..), getCurrentTime, utctDay)
+import Data.Time.Format (formatTime, defaultTimeLocale, iso8601DateFormat)
+
+
+firstHaskellRepoCreated :: Day
+firstHaskellRepoCreated = ModifiedJulianDay 58211
```

Comment:
Can you, please, add a comment to this constant what is this date in human calendar?

Reply:
And also, I'm so curious, what is the first Haskell library at GitHub?

Reply:
The first Haskell repo was created on 2008-04-03!

Reply:
@rashadg1030 Do you have a link to this repo? :slightly_smiling_face: